Dockerfile.arm32v7

```diff
@@ -0,0 +1,38 @@
+FROM alpine:latest AS builder
+
+RUN apk --update --no-cache add curl tar && \
+    curl -L https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-arm.tar.gz | tar zxvf - -C . && mv qemu-3.0.0+resin-arm/qemu-arm-static .
+
+FROM arm32v7/alpine:latest
+
+# Copy across the qemu binary that was downloaded in the previous build step
+COPY --from=builder /qemu-arm-static /usr/bin
+
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.docker.dockerfile="/Dockerfile" \
+      org.label-schema.license="MIT" \
+      org.label-schema.name="jnovack/docker-autossh" \
+      org.label-schema.url="https://hub.docker.com/r/jnovack/docker-autossh/" \
```

Comment:
This was my fault.  The correct URL is `https://hub.docker.com/r/jnovack/autossh/`